src/Entity/MetadataDisplayEntity.php

```diff
@@ -415,9 +415,9 @@ public function renderNative(array $context) {
   /**
    * {@inheritdoc}
    */
-  public function getTwigVariablesUsed() {
+  public function getTwigVariablesUsed(array $twigtemplate) {
```

Comment:
@aksm I believe this needs a different method. Let's discuss this tomorrow. Overriding the current one makes doing the opposite (what was done before) obscure as hell